src/index.js

```diff
@@ -1,9 +1,28 @@
 import './style.css';
 import { addIcons, toDos, add } from './modules/ui.js';
 
+const clearCompleted = () => {
+  const toDoList = JSON.parse(localStorage.getItem('allEntries'));
+  const complete = toDoList.filter((toDo) => toDo.completed === true);
+  const indexes = [];
+  for (let i = 0; i < complete.length; i += 1) {
+    const index = (complete[i].index) - 1;
+    indexes.push(index);
+  }
+
+  for (let i = indexes.length - 1; i >= 0; i -= 1) toDoList.splice(indexes[i], 1);
+
```

Comment:
- [ ] According to the requirements, you should use the `filter` method to remove all completed tasks from your to-do list. Kindly change the method used to do so from `splice` to `filter`. 🙏 👍 

![image](https://user-images.githubusercontent.com/77106962/206593873-e2adb64d-0414-4956-b64b-8e0c516e295d.png)